build_stats.sh

```diff
@@ -25,9 +25,10 @@ cd ..
 # Correct PATH and LD_LIBRARY_PATH variables.
 export PATH=${R_INST_DIR}/bin:${PATH}
 export LD_LIBRARY_PATH=${R_INST_DIR}/lib/R/lib:${LD_LIBRARY_PATH}
+export R_LIBS_USER=${R_INST_DIR}/lib/R/library/  # Needed by devtools::install
```

Comment:
If this variable is purely local to this script (which I think is the case), we don't need (and shouldn't use) `export`.